core/settings/project.py

```diff
@@ -73,3 +73,5 @@
 FARM_WORKSPACE = 'kartoza'
 FARM_LAYER_NAME = 'farm_portion'
 FARM_ID_COLUMN = 'id'
+
+IS_HEALTHYRIVERS = True
```

Comment:
please move this into docker compose envirotment instead. Make default as False.